apps/src/musicMenu/MusicMenu.jsx

```diff
@@ -163,6 +163,20 @@ const optionsList = [
       {value: 'tonejs', description: 'Use the ToneJS player (default).'},
     ],
   },
+  {
+    name: 'sort-cdo-sounds-by-type',
+    type: 'radio',
+    values: [
+      {
+        value: 'true',
+        description: 'Sort all Code.org sounds by type in the sounds panel.',
+      },
+      {
+        value: 'false',
+        description: 'Use the default sort for sounds in the sounds panel.',
+      },
+    ],
+  },
```

Comment:
Thanks for adding this.  Just a nit, but for recent additions, I've tried to put the `false` case first, and added `(default).` to the end of that option's description.